jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/comments/CaseCommentsPresenter.java

```diff
@@ -127,7 +122,11 @@ protected void addCaseComment(String caseCommentText) {
     protected void updateCaseComment(final CaseCommentSummary caseCommentSummary,
                                      String caseCommentNewText) {
         caseService.call(
-                (Void) -> refreshComments()
+            (Void) -> {
+                visibleCaseComments.remove(caseCommentSummary);
```

Comment:
Not sure I understood why it is needed to remove and add the caseCommentSummary object into the list. Perhaps only the text should be updated in the object and not even call the refreshComments method.